docs/CHANGELOG.rst

```diff
@@ -4,6 +4,35 @@ gufe Change Log
 
 .. current developments
 
+v1.3.0
+====================
+
+**Added:**
+
+* Added ExecutionInterrupt, a special exception that does not get handled as a ``ProtocolUnitFailure``.
+
+**Changed:**
+
+* Nodes and edges are now sorted by inchikey before being added to a networkx graph in the ``LigandNetwork.graph`` property. This replaces gufekey sorting which is not stable between versions and should result in reproducible network generation.
+* The message stating that partial charges are already present in an ``ExplicitMoleculeComponent`` is now included in ``logger.info``, rather than as a warning message. This should make output significantly less noisy for some users.
+* ``Protocol`` subclasses now require that the ``_settings_cls``
+  attribute is set to the intended ``SettingsbaseModel``
+  subclass. This attribute is validated during ``Protocol``
+  instantiation.
+* ``GufeTokenizable.from_json`` now falls back to loading ``dict`` representation if from ``keyed_chain`` fails
+* ``ExplicitMoleculeComponent`` now uses ``GufeTokenizable`` ``to_json`` and ``from_json`` methods via inheritance
+
+**Deprecated:**
+
+* ``Transformation.dump``, ``Transformation.load`` are now deprecated, use ``Transformation.to_json`` and ``Transformation.from_json`` instead
+
+**Fixed:**
+
+* Fixed bug where an error was only being raised if the difference between the sum of partial charges and the small molecule's net charge was a positive value. Behavior has been fixed such that negative discrepancies now raise an error as well.
+* Under some rare circumstances calling ``SmallMoleculeComponents.to_openff()`` may have lead to hydrogens being re-assigned when converted to OpenFF Molecules (e.g. during Protocol execution). ``SmallMoleculeComponents`` now explicitly pass the ``hydrogens_are_explicit=True`` flag on OpenFF Molecule creation to avoid this issue.
```

Comment:
```suggestion
* Under some rare circumstances calling ``SmallMoleculeComponents.to_openff()`` may have led to hydrogens being re-assigned when converted to OpenFF Molecules (e.g. during Protocol execution). ``SmallMoleculeComponents`` now explicitly pass the ``hydrogens_are_explicit=True`` flag on OpenFF Molecule creation to avoid this issue.
```